src/utils/utils.ts

```diff
@@ -179,7 +179,7 @@ export function getProperty (value: { [key: string]: any}, key: string): any {
   }
   
   const keys: string[] = key.split('.');
-  let result: any = value[keys.shift()];
+  let result: any = value[keys.shift()!];
```

Comment:
Indeed, I forgot that one.

Still getting use to all these new TS 2.0+ features.